shacl12-core/index.html

```diff
@@ -465,17 +471,17 @@ <h2>Document Outline</h2>
 				The introduction includes a <a href="#terminology">Terminology</a> section.
 			</p>
 			<p>
-				The sections 2 and 3 cover SHACL shapes, constraints, as well as property paths.
+				Sections 2 and 3 cover SHACL shapes and constraints, as well as property paths.
 			</p>
 			<p>
-				The section 4 introduces node expressions, while section 5 defines validation in SHACL.
+				Section 4 introduces node expressions, while section 5 defines validation in SHACL.
 			</p>
 			<p>
-				The section 6 defines the built-in SHACL Core constraint components. Meanwhile, section 7 discusses non-validating properties.
+				Section 6 defines the built-in SHACL Core constraint components, and section 7 discusses non-validating properties.
 			</p>
 			<p>
 				The syntax of SHACL is RDF.
-				The examples in this document use Turtle [[rdf12-turtle]], JSON-LD [[json-ld]] and SHACL-C.
+				The examples in this document use Turtle [[rdf12-turtle]], JSON-LD [[json-ld]], and SHACL-C [[shacl12-shacl-c]].
```

Comment:
The question mark I'm adding makes this reference informative a/k/a non-normative.
```suggestion
				The examples in this document use Turtle [[rdf12-turtle]], JSON-LD [[json-ld]], and SHACL-C [[?shacl12-shacl-c]].
```